### PR TITLE
Service Mesh Workshop Updates

### PR DESCRIPTION
**Pull request by dudash**
updates to the service mesh workshop

### CODE REVIEW

File: content/workshops/openshift_service_mesh/lab3.3_jaeger.md [hunk 1]
```diff
@@ -51,7 +52,7 @@ Let's inspect the traces.
 
 <blockquote>
 <i class="fa fa-desktop"></i>
-On the left bar under 'Search', select 'app-ui.microservies-demo' for 'Service' and 'boards-microservices-demo.svc.cluster.local'for 'Operation'.  
+On the left bar under 'Search', select 'app-ui.microservices-demo' for 'Service' and 'boards-microservices-demo.svc.cluster.local' for 'Operation'.
```
Comment (theckang) [inline content/workshops/openshift_service_mesh/lab3.3_jaeger.md:55]: Let's switch this to '{service-name}.userX' to be in sync with what workshop users are looking at.

File: content/workshops/openshift_service_mesh/lab3.3_jaeger.md [hunk 2]
```diff
@@ -138,7 +139,7 @@ for ((i=1;i<=5;i++)); do curl -s -o /dev/null $GATEWAY_URL/profile; done
 Inspect the traces.  
 <blockquote>
 <i class="fa fa-desktop"></i>
-On the left bar under 'Search', select 'app-ui.microservies-demo' for 'Service' and 'userprofile-microservices-demo.svc.cluster.local'for 'Operation'.  
+On the left bar under 'Search', select 'app-ui.microservices-demo' for 'Service' and 'userprofile-microservices-demo.svc.cluster.local' for 'Operation'.
```
Comment (theckang) [inline content/workshops/openshift_service_mesh/lab3.3_jaeger.md:142]: Let's switch this to '{service-name}.userX' to be in sync with what workshop users are looking at.

File: content/workshops/openshift_service_mesh/lab4.3_faultinjection.md [hunk 3]
```diff
@@ -184,7 +175,7 @@ Inspect the traces.
 
 <blockquote>
 <i class="fa fa-desktop"></i>
-On the left bar under 'Search', select 'app-ui.microservies-demo' for 'Service' and 'userprofile-microservices-demo.svc.cluster.local'for 'Operation'.
+On the left bar under 'Search', select 'app-ui.microservices-demo' for 'Service' and 'userprofile-microservices-demo.svc.cluster.local'for 'Operation'.
```
Comment (theckang) [inline content/workshops/openshift_service_mesh/lab4.3_faultinjection.md:178]: '{service-name}.userX'

File: content/workshops/openshift_service_mesh/lab1.3_deploymsa.md [hunk 4]
```diff
@@ -187,9 +66,9 @@ Deploy the context scraper service:
 ```
 oc new-app -f ./openshift-configuration/context-scraper-fromsource.yaml \
   -p APPLICATION_NAME=context-scraper \
-  -p NODEJS_VERSION_TAG=8-RHOAR \
+  -p NODEJS_VERSION_TAG=node-12 \
```
[the remark this answers is not in the document]
Comment (dudash) [reply content/workshops/openshift_service_mesh/lab1.3_deploymsa.md:69]: switching to 10 is fine - I'll do that.

File: content/workshops/openshift_service_mesh/lab1.3_deploymsa.md [hunk 5]
```diff
@@ -51,7 +51,7 @@ Deploy the boards service:
 ```
 oc new-app -f ./openshift-configuration/boards-fromsource.yaml \
   -p APPLICATION_NAME=boards \
-  -p NODEJS_VERSION_TAG=node-12 \
+  -p NODEJS_VERSION_TAG=node-10 \
```
Comment (theckang) [inline content/workshops/openshift_service_mesh/lab1.3_deploymsa.md:54]: Do I have a different tag setup than you?  I'm worried the buildconfig is pulling a different imagestream.  I've got 'nodejs:10' where the imagestream 'nodejs' has tags '10, 8, 8-RHOAR, latest'.

Comment (dudash) [reply content/workshops/openshift_service_mesh/lab1.3_deploymsa.md:54]: hmm. let me check.

File: content/workshops/openshift_service_mesh/lab5.3_prep.md [hunk 6]
```diff
@@ -31,15 +31,15 @@ Note: your admin username is `admin` and the password is autogenerated. These ar
 <i class="fa fa-terminal"></i> We configure via Kubernetes resources to create the realm + roles + clients & users as follows:
 </blockquote>
 ```
-oc apply -f ./keycloak-realm.yaml
+oc apply -f ./sso-realm.yaml
 ```
 
 ```
-oc apply -f ./keycloak-user-1.yaml
+oc apply -f ./sso-user-1.yaml
```
Comment (theckang) [inline content/workshops/openshift_service_mesh/lab5.3_prep.md:38]: `sso-user1.yaml`

File: content/workshops/openshift_service_mesh/lab5.3_prep.md [hunk 7]
```diff
@@ -31,15 +31,15 @@ Note: your admin username is `admin` and the password is autogenerated. These ar
...
 ```
 
 ```
-oc apply -f ./keycloak-user-2.yaml
+oc apply -f ./sso-user-2.yaml
```
Comment (theckang) [inline content/workshops/openshift_service_mesh/lab5.3_prep.md:42]: `sso-user2.yaml`

File: content/workshops/openshift_service_mesh/lab1.3_deploymsa.md (hunk 5 above)
Comment (dudash) [reply content/workshops/openshift_service_mesh/lab1.3_deploymsa.md:54]: yup - my bad, fixing to use the on that available by default in the RHPDS cluster image stream

**Pull request merged**